cmd/context/use.go

```diff
@@ -90,8 +90,8 @@ Or a Kubernetes context:
 
 func (c *ContextCommand) Run(ctx context.Context, ctxOptions *ContextOptions) error {
 	ctxStore := okteto.ContextStore()
-	ctxOptions.initFromContext()
 	ctxOptions.initFromEnvVars()
+	ctxOptions.initFromContext()
```

Comment:
We were initialising always from the env vars so only `OKTETO_TOKEN` could be replaced